app/uk/gov/hmrc/economiccrimelevyregistration/views/BusinessNameView.scala.html

```diff
@@ -38,7 +38,7 @@ <h1 class="govuk-heading-l">@HtmlFormat.escape(messages("businessName.heading"))
         @govukInput(
             InputViewModel(
                 field = form("value"),
-                label = LabelViewModel(Text("businessName.heading")).asHiddenPageHeading()
+                label = LabelViewModel(Text(messages("businessName.heading"))).asHiddenPageHeading()
```

Comment:
Does this and the others create an additional H1 tag? If it does, it'll fail accessibility.